apps/test/code-studio/ProgressDotTest.js

```diff
@@ -60,14 +60,6 @@ describe('ProgressDot component tests', () => {
     expect(radiumHelper(result.props.children[0].props.style).borderColor).to.equal(color.lighter_gray);
   });
 
-  it('bubble interior renders level title if not on overview page and not showing an icon', () => {
-    const result = ReactTestUtils.renderIntoDocument(
-      <BubbleInterior title={level.title} courseOverviewPage={false} showingIcon={false} showingLevelName={true} />
-    );
-
-    expect(ReactDOM.findDOMNode(result).innerHTML).to.equal('1');
-  });
-
```

Comment:
Brent: Make sure you removed the right test, and also remove the `courseOverViewPage` prop from these tests, as it no longer is used.